src/main/java/ca/on/oicr/pineryreports/reports/impl/GazpachoProjectStatusReport.java

```diff
@@ -295,6 +299,10 @@ protected void collectData(PineryClient pinery) throws HttpResponseException, IO
     List<SampleDto> allProjectSamples = allSamples.stream().filter(byProject(project)).collect(Collectors.toList());
     Map<String, SampleDto> allSamplesById = mapSamplesById(allSamples); // separate since hierarchy may include multiple projects
     List<RunDto> allRuns = pinery.getSequencerRun().all();
+    instrumentsById = pinery.getInstrument().all().stream()
+        .collect(Collectors.toMap(InstrumentDto::getId, dto -> dto));
```

Comment:
Can use `Function::identity` rather than `dto -> dto`

Reply:
Strangely enough, I can't -- it gives a compilation error.